CMakeLists.txt

```diff
@@ -284,7 +284,7 @@ if(WITH_CORE)
 
   set (WITH_PDAL FALSE CACHE BOOL "Determines whether PDAL support should be built")
   if(WITH_PDAL)
-    set(HAVE_PDAL TRUE)
+    set(HAVE_PDAL_QGIS TRUE) # note -- we can't use HAVE_PDAL here as the grass public headers redefine this!
```

Comment:
urelated to your change but the duplication of this line with line 437 is confusing (to me at least). I bet this line should be suppressed. HAVE_PDAL_QGIS should only be set if WITH_PDAL is set and PDAL_FOUND, which is the condition triggering line 437.
The same applies for the set(HAVE_EPT TRUE) at line 292

Reply:
Yes, you're right! I'll fix

Reply:
Fixed